Life as a teacher description update (#4202)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher.md

---
title: "Life as a teacher"

> description: Learn how rewarding teaching can be, including pay, career progression, support for early career teachers and career changers, and the subjects and pupils you could teach.

Author: ```suggestion
description: Explore the rewards of teaching, including pay, career growth, support, and the range of subjects and pupils you could teach.
```

> description: Wondering if teaching is the right career for you? Learn about teacher pay and pensions, career progression, support for early career teachers and career changers, and discover the subjects and age groups you could teach.  

Author: ```suggestion
description: Learn how rewarding teaching can be, including pay, career progression, support for early career teachers and career changers, and the subjects and pupils you could teach.
```
